cmd/vicadmin/vicadm.go

```diff
@@ -290,21 +294,35 @@ func listVMPaths(ctx context.Context, s *session.Session) ([]url.URL, error) {
 
 	log.Infof("Found %d candidate VMs in resource pool %s for log collection", len(children), ref.String())
 
-	directories := []url.URL{}
+	logfiles := []logfile{}
 	for _, child := range children {
 		path, err := child.DSPath(ctx)
+
 		if err != nil {
 			log.Errorf("Unable to get datastore path for child VM %s: %s", child.Reference(), err)
 			// we need to get as many logs as possible
 			continue
 		}
 
+		logname, err := child.Name(ctx)
```

Comment:
Yes we have had previous issues with vm renaming as well. Should that be covered in this pr?